### PR TITLE
CommentSourcePointer-move-methods-down 

### PR DESCRIPTION
**Pull request by MarcusDenker**
to prepare moving the commentSourcePointer ivar down to class (as it is nil for MetaClass), this PR

- adds forwarding methods to MetaClass
- move methods down to Class

### CODE REVIEW

File: src/Kernel/Class.class.st [hunk 1]
```diff
@@ -313,6 +313,50 @@ Class >> classBuilder [
...
+Class >> classComment: aString stamp: aStamp [
+	"Store the comment, aString or Text or RemoteString, associated with the class we are organizing.  Empty string gets stored only if had a non-empty one before."
+
+	| pointer header oldComment oldStamp preamble |
+	oldComment := self comment.
+	oldStamp := self commentStamp.
+
+	aString string = oldComment string ifTrue: [ ^ self ].
+
+	pointer := self instanceSide commentSourcePointer ifNil: [0].
```
Comment (jecisc) [inline src/Kernel/Class.class.st]:
```suggestion
	pointer := self commentSourcePointer ifNil: [0].
```

IIUC the goal, here it should always be equivalent to self?

File: src/Kernel/Class.class.st [hunk 2]
```diff
@@ -313,6 +313,50 @@ Class >> classBuilder [
...
+				nextPutAll: ' commentStamp: '.
+			aStamp storeOn: strm.
+			strm nextPutAll: ' prior: '; nextPutAll: pointer printString ].
+		file nextChunkPut: header; cr ].
+
+	SourceFiles
+		writeSource: aString
+		preamble: preamble
+		onSuccess: [ :newSourcePointer |
+			self instanceSide commentSourcePointer: newSourcePointer]
```
Comment (jecisc) [inline src/Kernel/Class.class.st]:
```suggestion
			self commentSourcePointer: newSourcePointer]
```

Same

File: src/Kernel/Class.class.st [hunk 3]
```diff
@@ -406,6 +450,45 @@ Class >> classVariablesNeedFullDefinition [
...
+    ^ self commentSourcePointer
+		  ifNil: [ '' ]
+		  ifNotNil: [ :ptr | SourceFiles sourceCodeAt: ptr ]
+]
+
+{ #category : #'accessing - comment' }
+Class >> comment: aStringOrText [
+	"Set the receiver's comment to be the argument, aStringOrText."
+
+	self instanceSide classComment: aStringOrText
```
Comment (jecisc) [inline src/Kernel/Class.class.st]:
```suggestion
	self classComment: aStringOrText
```

**Review by jecisc — APPROVED** [review]

**Comment by jecisc** [comment]
The global changes is good. I suggested some code cleanings as a bonus

File: src/Kernel/Class.class.st (hunk 1 above)
Comment (MarcusDenker) [reply src/Kernel/Class.class.st]: yes, I did not look at the methods yet as they should be deprecated in a next step, but of course we can do this change now already

**Pull request merged**